spec/unit/filters/humanized_spec.rb

```diff
@@ -1,20 +1,46 @@
 require 'rails_helper'
 
 describe ActiveAdmin::Filters::Humanized do
-  let(:param) { ['category_id_eq', '1'] }
-
-  subject { ActiveAdmin::Filters::Humanized.new(param) }
+  let(:category) { Category.create!(name: 'Jazz') }
 
   describe '#value' do
-    it 'should equal query string parameter' do
-      expect(subject.value).to eq('1')
+    context 'parameter points to a related model' do
+      it 'should equal the :name of category instance' do
+        param = ['category_id_eq', category.id]
+        humanizer = ActiveAdmin::Filters::Humanized.new(param)
+        expect(humanizer.value).to eq('Jazz')
+      end
+    end
+
+    context 'parameter looks like a related model but does not' do
+      it 'should equal query string parameter' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.